docs/firmware-update.md

```diff
@@ -48,5 +48,3 @@ If the led is not blinking:
      - Where /dev/ttyUSB0 is my serial port
 
 If the instructions are not clear or sound a bit complicated, please wait a bit more for us to create a Ping-Viewer version that allows you to recover devices.
-
-> TODO explain bootloader process
```

Comment:
An issue should be created to the product documentation for that.

Reply:
https://github.com/bluerobotics/ping-viewer/issues/764